src/Evolution/Systems/GrMhd/ValenciaDivClean/Subcell/TciOnDgGrid.hpp

```diff
@@ -41,16 +46,16 @@ namespace grmhd::ValenciaDivClean::subcell {
  *   negative density and the cell is troubled.
```

Comment:
negative (or extremely small)

Also maybe add a note (at the end ?) that if this tci_option is approximately equal to or larger than the atmosphere_density, the atmosphere will be flagged as troubled